modules/nw-metallb-when-metallb.adoc

```diff
@@ -10,7 +10,9 @@ Using MetalLB is valuable when you have a bare-metal cluster, or an infrastructu
 
 You must configure your networking infrastructure to ensure that network traffic for the external IP address is routed from clients to the host network for the cluster.
 
-After deploying MetalLB with the MetalLB Operator, when you add a service of type `LoadBalancer`, MetalLB provides a platform-native load balancer.
+After deploying MetalLB with the MetalLB Operator, when you add a service of type `LoadBalancer`, MetalLB provides a platform-native load balancer. 
+
+When external (ingress) traffic enters your {product-title} cluster through a MetalLB LoadBalancer service, MetalLB configures the Source Network Address Translation (SNAT) for the return (egress) traffic to use the service’s IP address. This ensures that when the traffic exits the cluster back to the client, it appears to come from the service’s external IP address rather than the node IP address.
```

Comment:
technically, it's not metallb to do this. And it's a bit more complex than that (because it's a reply on a session with a dnat towards the clusterip). I don't think it's important in this scenario. Can we just mention that the return traffic will have the loadbalancer ip as source ip without going into details on how this is done?